docs/sphinx/source/whatsnew/v0.3.1.txt

```diff
@@ -15,8 +15,10 @@ Enhancements
 Bug fixes
 ~~~~~~~~~
 
+* Fixes snlinverter Pnt problem (:issue:`140`)
```

Comment:
I know I asked for brief, but maybe something more like "Fixes night tare issue in snlinverter. The night tare was erroneously activated when the AC output power, rather than the DC input power, was less than the inverter's self-consumption" or similar. I had to look up Pnt, so I'm sure others would need to as well.